specification/web/resource-manager/Microsoft.Web/stable/2022-09-01/StaticSites.json

```diff
@@ -1034,10 +1064,10 @@
         ],
         "responses": {
           "200": {
-            "description": "OK.",
-            "schema": {
-              "$ref": "#/definitions/StaticSiteUserProvidedFunctionAppsCollection"
-            }
+            "description": "OK."
+          },
```

Comment:
I think technically this change would require a new api version. Was your server already returning it in practice though? Or is this an unreleased ap-version??

Reply:
So on this point I wanted to clarify a couple things

1. I'm not sure how to improve this, but I think the diff displayed here is a bit misleading. So far, I have only made additions to this file for the new database connections APIs (no removals); however, since they follow a similar format to other APIs, I think the diff tool is displaying it as replacing the user provided function app APIs with the database connection APIs and then also adding the user provided function app APIs. Let me know if this was your concern and if this clears anything up, I'm not intending on changing any of the existing APIs.

2. This 2022-09-01 api version I believe is a new, unreleased api version anyways. My PR is made against our broader team's feature branch.

Reply:
Could you clarify what exactly is a breaking change here? I am adding new APIs related to databaseConnections, which is a new set of APIs we'll be supporting. To clarify again, there's no change to the existing APIs, I believe it's just an artifact of the diffing tool (for example if you look at the API for `/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Web/staticSites/{name}/builds/{environmentName}/userProvidedFunctionApps` which seems to have been removed, it actually appears around line 1380 in the new file).